### PR TITLE
Added support for pino-http and hapi-pino

### PR DESCRIPTION
**Pull request by mcollina**
As titled :).

I've done a bunch of changes, let me know if you want things to be handled differently. In pino-http there is no "content length" concept, but I hope to add it there asap.

cc @davidmarkclements

### CODE REVIEW

**Comment by davidmarkclements** [comment]
extra context - pino-http support is also express-pino-logger and express-koa-logger support

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ npm install pino-colada
 - [merry](https://github.com/yoshuawuyts/merry)
 - [garnish](https://github.com/mattdesl/garnish)
 - [@studio/log](https://github.com/javascript-studio/studio-log)
+- [pino-http](https://github.com/pinojs/pino-http)
```
Comment (davidmarkclements) [inline README.md:30]: should (may) we add hapi-pino too (and also express-pino-logger, koa-pino-logger ?)

**Review by davidmarkclements — COMMENTED** [review]
would be nice to have express-pino-logger example too

File: README.md (hunk 1 above)
Comment (mcollina) [reply README.md:30]: I've added hapi-pino, I won't add the others as it might be too much.

**Comment by lrlna** [comment]
Ohhhh this is rad @mcollina!!

I am going to look into it a bit more tomorrow cause I am still sick in bed
today, but on the first glance ✌️✌️✌️

File: example-hapi-pino.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const Hapi = require('hapi')
```
Comment (lrlna) [inline example-hapi-pino.js]: would you mind if we keep the examples in ES3 for consistency?

File: README.md (hunk 1 above)
Comment (lrlna) [reply README.md:30]: yea that'd probably be too many; `pino-http` and `hapi-pino` are good additions tho 👌

**Comment by lrlna** [comment]
okay, had a chance to properly look through! All looks good; just a little comment on one of the examples ^___^

File: example-hapi-pino.js (hunk 2 above)
Comment (mcollina) [reply example-hapi-pino.js]: Done :).

**Comment by mcollina** [comment]
@lrlna updated :)

**Comment by lrlna** [comment]
meow-excellent! ✨

**Comment by lrlna** [comment]
`v1.2.3`!

**Pull request merged**